templates/catalog/edit.html

```diff
@@ -0,0 +1,134 @@
+{% extends "layout.html" %}
+{% block bigcontainerclass %}white{% endblock %}
+
+{% block scripts %}
+
+<script>
```

Comment:
I fixed this by deleting the autocomplete function, and testing the existence of the domain using JavaScript. 
This however made it impossible to modify the domain name if it was written for example in capital letter or vice-versa (kdd=KDD). 
Perhaps, we can add a JS function to transform the text value into for example first capital letter of each word(complex networks to  Complex Networks)  when adding a paper !

Reply:
A good idea, but I cannot say that it will be 100% ok to silently capitalize all first letters.
Maybe, when we test for the existence we should all domains except the current ?

Reply:
Okay, I'll work on it